### ISSUE TITLE
move zip file generation to backend

### ISSUE THREAD

**Issue by polamoros** [winglang/wing#1701]
### Community Note

> Please vote by adding a 👍 reaction to the issue to help us prioritize.
> If you are interested to work on this issue, please leave a comment.

### Feature Spec

Zip file generation is done in the backend of the Console app.

### Use Cases

Download multiple files from a bucket

### Implementation Notes

use [archiver](https://www.npmjs.com/package/archiver)

### Component

Wing Console

**Comment by ainvoner** [comment]
not sure it is relevant anymore. @polamoros  if need to keep open, remove `stale` and reopen

**Comment by polamoros** [comment]
@ainvoner yes, we should move [this code](https://github.com/winglang/wing/blob/12779b7d9d946db3b91809794967b8a5b18162e4/apps/wing-console/console/ui/src/shared/use-download-file.ts#L54) to the BE
